### DISCUSSION TITLE
Tshock Terminal is messing up while someone is online

### DISCUSSION THREAD

**Discussion by Netroshin1** [Pryaxis/TShock#2901] [Q&A] [answered]
Hello, 
I have a problem with my terminal while some players are on my server.
Everything seems okay if no one is connected. I can type commands and the result will be displayed as usual.
If someone is connected to the server, the output text is cut off of the terminal and I can't see what I am typing in this moment. In addition to that a line will be "shown" every 10 seconds in the Terminal. My screenlog shows me the whole line:

k1/20 on DownTown @ 0.0.0.0:1400 (TShock for Terraria v5.1.3.0)

When every player disconnects I can type something in the terminal. There isn't the whole output shown but after that it will display anything as usual.

Here is my Terminal output:

> ![terminal](https://user-images.githubusercontent.com/111142464/213299268-394cd5ce-d2f5-4be3-b87f-f21139302f4f.PNG)

Here is my Screenlog of the same Terminal output shown on top (the marked lines were cut off):

> ![screenlog](https://user-images.githubusercontent.com/111142464/213298984-92d82667-4c05-4b78-84ed-8319bc2cf0d8.PNG)

What is happening? Why is this happening? 
I hope someone can help me. Thanks a lot!

**Comment by sgkoishi** [comment, marked as answer]
See #2820 #2889.
